WordPress/wordpress-playground
# Add Gutenberg PR previewer page

## What?

A super simple HTML form to preview Gutenberg PRs and potentially replace gutenberg.run. It uses Blueprints instead of `?gutenberg-pr=<number>` so that other projects can simply view the page source and build their own version.

@aduth is it okay if I reuse the original design?

<img width="760" alt="CleanShot 2023-06-01 at 18 41 50@2x" src="https://github.com/WordPress/wordpress-playground/assets/205419/a96289f7-1666-48da-af40-670ff17a9d42">

## Testing Instructions

1. Run `npm run dev`
2. Go to http://localhost:5400/gutenberg.html
3. Paste a Gutenberg PR number or link
4. Submit the form
5. Confirm the PR got downloaded and applied

cc @dd32 @youknowriad @dmsnell
